bower.json

```diff
@@ -6,6 +6,7 @@
   },
   "devDependencies": {
     "requirejs": "~2.1.9",
+    "immutable": "~3.6.2",
```

Comment:
Is this needed? We didn't have to add this to noting, it got installed with scribe.